package.json

```diff
@@ -22,9 +22,9 @@
         "url": "https://github.com/Microsoft/vscode-jupyter-powertoys/issues"
     },
     "qna": "https://github.com/microsoft/vscode-jupyter-powertoys/discussions",
-    "version": "0.0.5",
+    "version": "0.0.6",
     "engines": {
-        "vscode": "^1.67.0"
+        "vscode": "^1.70.0"
```

Comment:
I think this is the version needed (current stable) for the API changes that we picked up with notebook edit. Might have been one version before that, but this seemed safer.